backend/entities/ContactMethodType.cs

```diff
@@ -38,4 +38,13 @@ public ContactMethodType(string id, string description) : base(id, description)
         }
         #endregion
     }
+
+    public static class ContactMethodTypes
```

Comment:
You could use an Enum here instead even with string values.  We had originally some nice extensions to do string comparisons with them.

Reply:
For this usage (essentially just grouping some constants) I don't see the benefit of an enum.